.changeset/mean-mugs-suffer.md

```diff
@@ -0,0 +1,20 @@
+---
+"@fluidframework/container-definitions": major
+"@fluidframework/container-loader": major
+"@fluidframework/container-runtime": major
+"@fluidframework/datastore": major
+"@fluidframework/driver-definitions": major
+"@fluidframework/driver-utils": major
+"fluid-framework": major
+"@fluidframework/odsp-doclib-utils": major
+"@fluidframework/odsp-driver": major
+"@fluidframework/odsp-driver-definitions": major
+"@fluidframework/routerlicious-driver": major
+"@fluidframework/telemetry-utils": major
+"@fluid-private/test-end-to-end-tests": major
+"@fluidframework/tinylicious-client": major
+---
+
+Deprecate a number of error related enums, and replace them with new object-based based enumerations.
+
+The changes involve deprecation of error related enums ContainerErrorType, DriverErrorType, OdspErrorType and RouterliciousErrorType and replacement to new object-based enumerations of ContainerErrorTypes, DriverErrorTypes, OdspErrorTypes and RouterliciousErrorTypes. In addition, it adds a stricter type checking logic to the test suites on the changed error enumerations.
```

Comment:
Nit: these changesets are intended for external consumers, so I don't think we need to call out test changes here.
```suggestion
The changes involve deprecation of error related enums ContainerErrorType, DriverErrorType, OdspErrorType and RouterliciousErrorType and replacement to new object-based enumerations of ContainerErrorTypes, DriverErrorTypes, OdspErrorTypes and RouterliciousErrorTypes.
```

Reply:
I think the changeset should just say "Many deprecated classes are now removed. Refer to release notes of version <some_version> for the replacements." Or actually, I think this is one of the changes that we failed to announce before, so maybe the guidance on what to do to move away from the deprecated objects might have to go here.

But mainly my point is that this change doesn't deprecate anything; it removes things that were deprecated in the past.

And I agree with Josh's point about the tests, not necessary for release notes.